index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfoli</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="navbar">
+       <a class="logo">My Logo</a>
+       <img src="Union.png" alt="menubar">
+    </header>
+  
+     <section class="main">
+       <h1 class="header">I’m Ryisa Glad to see you! </h1>
+          <p class="section">
+           I’m a software developer! I can help you build a product , feature or 
+           website Look through some of my work and experience! If you like what 
+           you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+           <p class="footer">LET CONNECT</p>
+            <div class="img">
```

Comment:
- [ ] As mentioned above about semantic HTML kindly change this to an `<ul></ul>` instead of a `<div></div>`